experimental/unicodeset_parser/src/lib.rs

```diff
@@ -7,7 +7,9 @@
 //! This crate provides parsing functionality for [UTS #35 - Unicode Sets](https://unicode.org/reports/tr35/#Unicode_Sets).
 //! Parses into [`CodePointInversionListAndStringList`](icu_collections::codepointinvliststringlist::CodePointInversionListAndStringList).
 //!
-//! See [`parse`](crate::parse) for more information.
+//! See [`parse`](crate::parse()) for more information.
+//!
+//! [`ICU4X`]: ../icu/index.html
```

Comment:
Not sure this works on docs.rs

Reply:
Pre-existing code that I copied from some other component.